src/i18n/locales/en/translation.json

```diff
@@ -288,6 +288,39 @@
         "errors": {
           "connection_failed": "Connection failed. Is the BTCPay connection URL correct and accessible?"
         }
+      },
+      "commando": {
+        "title": "Core Lightning",
+        "description": "Connect to your Core Lightning node",
+        "page": {
+          "title": "Connect to your Core Lightning node",
+          "instructions": "Make sure you have Core Lightning version 0.12.0 or newer, the commando plugin is running and your node is accessible over the Lightning Network. Create a rune by running 'lightning-cli commando-rune'."
```

Comment:
can we add a guide/help entry on how this is done exactly?

Reply:
In my opinion mostly technical people will use it anyway so I would do this later. My video on how to set it up on Digital Ocean can be published as a guide though.

Reply:
ok, though raspiblitz people will do that for example